From c06da9254671694e1a1aa0f5c964a61dc7b211ff Mon Sep 17 00:00:00 2001
From: Martino Facchin <m.facchin@arduino.cc>
Date: Tue, 30 Jul 2019 11:47:04 +0200
Subject: [PATCH 01/17] FlashIAPBlockDevice: add missing mbed namespace

---
 .../blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h b/components/storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h
index 98cd805612..ec64c50616 100644
--- a/components/storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h
+++ b/components/storage/blockdevice/COMPONENT_FLASHIAP/FlashIAPBlockDevice.h
@@ -133,7 +133,7 @@ public:
     *  @param size     Size to erase in bytes
     *  @return         True if erase is valid for underlying block device
     */
-    virtual bool is_valid_erase(bd_addr_t addr, bd_size_t size) const;
+    virtual bool is_valid_erase(mbed::bd_addr_t addr, mbed::bd_size_t size) const;
 
 
 private:
-- 
2.29.2

