From 54ef736ead31df4ef2520642724cd62612a6332b Mon Sep 17 00:00:00 2001
From: George Beckstein <donbeckstein@live.com>
Date: Fri, 3 Apr 2020 06:10:53 -0400
Subject: [PATCH 17/17] Changed pin names in example to be more universal

---
 drivers/I2CSlave.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/I2CSlave.h b/drivers/I2CSlave.h
index 3087a23538..663b326870 100644
--- a/drivers/I2CSlave.h
+++ b/drivers/I2CSlave.h
@@ -53,7 +53,7 @@ namespace mbed {
  * #error [NOT_SUPPORTED] I2C Slave is not supported
  * #endif
  *
- * I2CSlave slave(p3, p4);
+ * I2CSlave slave(I2C_SDA, I2C_SCL);
  *
  * int main() {
  *
@@ -86,7 +86,7 @@ namespace mbed {
  *
  * // Master side of the example
  *
- * I2C master(p3, p4);
+ * I2C master(I2C_SDA, I2C_SCL);
  *
  * static const char* to_send[] = { "abcde", "12345", "EFGHI" };
  *
-- 
2.29.2

